.github/matrix.json

```diff
@@ -0,0 +1,3 @@
+{
+  "ruby": ["2.7", "3.0"]
```

Comment:
jenkins also runs 3.1, should we add that?

Reply:
Yes.

Reply:
given we decided this file only indicates "minimal tested versions", I think 3.1 doesn't (yet) belong in here